gRPC/BICgRPCServer/ClassesSource/BICListener.cpp

```diff
@@ -126,11 +126,15 @@ namespace BICGRPCHelperNamespace
     /// <param name="id"></param>
     void BICListener::onLastStimulationFunctionId(const uint16_t id)
     {
-        if (m_isStimulating)
+        /*try
```

Comment:
what's going on here? if statement removed and the rest is commented out?

Reply:
Commented out during testing and forgot to uncomment